docs/checkers/index.md

```diff
@@ -1809,6 +1809,30 @@ class SecretKeeper:
         return self._secret == secret
 ```
 
+(E3701)=
+
+### Invalid Field Call (E3701)
```

Comment:
use lowercase letters for "field" and "call" (i.e., only capitalize the first word in the subsection titles)